src/main/java/com/Commands.java

```diff
@@ -27,4 +27,18 @@ private static String getStringParameter(String parameter) {
         Scanner reader = new Scanner(System.in);
         return reader.nextLine();
     }
-}
\ No newline at end of file
+
+    public static MedicalSpecializiation getMedicialSpecializiation() {
+        System.out.println("Enter Doctor's medicial specializiation (PATHOLOGY, CARDIOLOGY, UROLOGSY, NEUROLOGY):");
+        String selectedMedicialSpecializiation = getStringParameter("Your choise:");
+        if (selectedMedicialSpecializiation.equals("PATHOLOGY")) {
+            return MedicalSpecializiation.PATHOLOGY;
+        } else if (selectedMedicialSpecializiation.equals("CARDIOLOGY")) {
+            return MedicalSpecializiation.CARDIOLOGY;
+        } else if (selectedMedicialSpecializiation.equals("UROLOGSY")) {
+            return MedicalSpecializiation.UROLOGY;
+        } else if (selectedMedicialSpecializiation.equals("NEUROLOGY")) {
```

Comment:
It redundant to return twice neurology. Leave only the last one: else return NEUROLOGY